RELEASENOTES.md

```diff
@@ -1,18 +1,45 @@
+This document contains the release notes for `rippled`, the reference server implementation of the Ripple protocol. To learn more about how to build, run or update a `rippled` server, visit https://xrpl.org/install-rippled.html
 
-> **Do you work at a digital asset exchange or wallet provider?** 
-> 
-> Please [contact us](mailto:support@ripple.com). We can help guide your integration.
+ 
+Have new ideas? Need help with setting up your node? Come visit us [here](https://github.com/ripple/rippled/issues/new/choose)
 
-## Updating `rippled`
+# Releases
 
-If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `yum`](https://ripple.com/build/rippled-setup/#updating-rippled). For other platforms, please [compile from source](https://wiki.ripple.com/Rippled_build_instructions).
+## Version 1.5.0
+
+The `rippled` 1.5.0 release introduces several improvements and new features, including support for gRPC API, API versioning, UNL propagation via the peer network, new RPC methods `validator_info` and `manifest`, augmented `submit` method, improved `tx` method, improved CLI parsing, improved protocol-level handshaking protocol, improved package building and various other minor bug fixes and improvements.
+
+This release also introduces two new amendments: `fixQualityUpperBound` and `RequireFullyCanonicalSig`.
+
+Several improvements to the sharding system are currently being evaluated for inclusion into the upcoming 1.6 release of `rippled`. These changes are incompatible with shards generated by previous versions of the code. 
+Additionally, an issue with the existing sharding engine can result in a server running versions 1.4 or 1.5 of the software to experience a deadlock and automatically restart when running with the sharding feature enabled. 
+At this time, the developers recommend running with sharding disabled, pending the improvements scheduled to be introduced with 1.6. For more information on how to disable sharding, please visit https://xrpl.org/configure-history-sharding.html
+
+ 
+**New and Updated Features**
+- Native [gRPC API](https://grpc.io/) API support. Currently, this API provides a subset of the full `rippled` [API](https://xrpl.org/rippled-api.html). You can enable the gRPC API on your server with a new configuration stanza. 
+- API Versioning which allows for future breaking change of RPC methods to co-exists with existing versions.
+- Nodes now receive and broadcast UNLs over the peer network under various conditions. see `rippled` [#3072](https://github.com/ripple/rippled/pull/3072)
+- Augmented `submit` method to include additional details on the status of the command.
+- Improved `tx` method to allow the search to specify a range of ledgers using the `min_ledger` and `max_ledger` fields in the request.
```

Comment:
Nit: I'm not sure that this sentence is correct. See the conversation here: https://github.com/ripple/xrpl-dev-portal/pull/809#discussion_r399539244. The semantics of `min_ledger` and `max_ledger` are kind of confusing (the method only uses those parameters for error handling; they don't affect the search). The semantics are a bit messy to put in the release notes, but I don't want the release notes to be misleading. So it's up to you whether you want to change this sentence or leave as is.

Reply:
Updated to correctly describe the improvement.